packages/renderer/src/stores/cli-tools.ts

```diff
@@ -0,0 +1,52 @@
+/**********************************************************************
+ * Copyright (C) 2023 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import { writable, type Writable } from 'svelte/store';
+import { EventStore } from './event-store';
+import type { CliToolInfo } from '../../../main/src/plugin/api/cli-tool-info';
+
+const windowEvents: string[] = ['extensions-started', 'cli-tool-create', 'cli-tool-remove'];
+const windowListeners = ['system-ready'];
```

Comment:
```suggestion
const windowListeners = ['system-ready', 'extensions-already-started'];
```

I think it should track also when an extension is stopped or started

```ts
windowEvents = ['extension-stopped', 'extensions-started'];
```

Reply:
in current implementation there are specific events for creating and removing cli tools from store. When extension started/stopped the store is already populated/disposed.